pkg/wireguard/agent/agent.go

```diff
@@ -45,7 +45,8 @@ import (
 )
 
 const (
-	listenPort = 51871
+	listenPort        = 51871
+	errOpNotSupported = unix.Errno(0x5f)
```

Comment:
Maybe we can use EOPNOTSUPP from pkg/syscall?

Reply:
The `syscall` package is deprecated and should not be used in Cilium anymore: https://github.com/cilium/cilium/issues/10116

I haven't found a alternative definition to `EOPNOTSUPP` in our tree :(

Reply:
I think it should be in `golang.org/x/sys/unix`. @tklauser might now better.

Reply:
Yup, it's in our vendored `golang.org/x/sys/unix`:

https://github.com/cilium/cilium/blob/99c129c99bc611d638be5ce2083a4135ff3bd61f/vendor/golang.org/x/sys/unix/zerrors_linux_amd64.go#L574

Also note that it doesn't have the same value `0x5f` on all platforms. Though, this is not really of relevance here given Cilium only officially supports `amd64` and `arm64`. These both use the same value.

Reply:
Huh. Thanks for digging, somehow my IDE did not find this (I guess because it was looking in `zerrors_linux.go` (not the `amd64` one). Will change!

Reply:
Yeah, it's kind of confusing. `zerrors_linux.go` only contains consts which are the same across all platforms, while `zerrors_linux_$GOARCH.go` contains consts which are different between (some) platforms.